node-module-system/calculate.js

```diff
@@ -0,0 +1,26 @@
+import sum from './add.js';
+import difference from './subtract.js';
+import product from './multiply.js';
+import quotient from './divide.js';
+
+const firstArg = Number(process.argv[2]);
+const secondArg = Number(process.argv[4]);
+
+switch (process.argv[3]) {
+  case 'plus':
+  case '+':
```

Comment:
Clever. Did you discover why the exercise has you type out the word `plus`, etc? It's because entering these special symbols on the command line is somewhat awkward -- they often need to be wrapped in quotes.